src/Coordination/KeeperDispatcher.cpp

```diff
@@ -542,6 +547,13 @@ void KeeperDispatcher::updateConfigurationThread()
                 continue;
             }
 
+            if (server->isRecovering())
+            {
+                LOG_INFO(log, "Server is recovering, will not apply configuration until recovery is finished");
+                std::this_thread::sleep_for(std::chrono::milliseconds(5000));
```

Comment:
Ok, but maybe a slightly smaller timeout?

Reply:
I went for the same value as for the init, it made sense to me to have it a bit larger because the process is basically reinitializing the quorum.
What would you say is a reasonable amount?